frontend/src/MovieFile/Editor/MovieFileEditorRow.js

```diff
@@ -105,7 +105,7 @@ class MovieFileEditorRow extends Component {
     return (
       <TableRow>
         <TableRowCell
-          className={styles.relativePath}
+          className={styles.title}
```

Comment:
I'd say change it back here and fix it on the css side. That way it jives with property name.

Reply:
It doesn't jive with the UI is why i swapped it lol.. But sure.